kernel/kselftest.py

```diff
@@ -177,7 +177,7 @@ def test(self):
         kself_args = self.params.get("kself_args", default='')
         if self.comp == "bpf":
             self.bpf()
-        if self.comp == "cpufreq":
+        elif self.comp == "cpufreq":
```

Comment:
this will break another part of the code like `result` is an undefined variable, please take care of that well, easy way is to return value from `cpufreq` function and store in result variable